source/crud/tests/write/bulkWrite.yml

```diff
@@ -332,3 +332,41 @@ tests:
                 data:
                     - {_id: 2, x: 23 }
                     - {_id: 3, x: 33 }
+    -
+        description: "BulkWrite continue-on-error behavior with unordered"
+        operation:
+            name: "bulkWrite"
+            arguments:
+                requests:
+                    -
+                        name: "insertOne"
+                        arguments:
+                            document: { _id: 2, x: 22 }
+                    -
+                        name: "insertOne"
+                        arguments:
+                            document: { _id: 3, x: 33 }
+                    -
+                        name: "insertOne"
+                        arguments:
+                            document: { _id: 4, x: 44 }
+                options: { ordered: false }
+        outcome:
+            error: true
```

Comment:
I just realized that [tests/README.rst](https://github.com/mongodb/specifications/blob/master/source/crud/tests/README.rst) would benefit from having this defined. I'll make that change and incorporate it into this PR soon.

Reply:
I've updated the test format descriptions in a later commit within this PR.